libs/wingsdk/test/shared-aws/counter.inflight.test.ts

```diff
@@ -55,6 +55,44 @@ test("inc(5)", async () => {
   expect(response).toEqual(prevValue); // returns previous value
 });
 
+test("key inc(1)", async () => {
+  // GIVEN
+  const prevValue = 887;
+  setupIncMock({
+    expectedTableName: MOCK_TABLE_NAME,
+    expectedAmount: 1,
+    initial: 0,
+    responseValue: prevValue + 1,
+    key: "my-key",
+  });
+
+  // WHEN
+  const client = new CounterClient(MOCK_TABLE_NAME);
+  const response = await client.inc(undefined, "my-key");
+
+  // THEN
+  expect(response).toEqual(prevValue); // returns previous value
```

Comment:
Why prevValue and not prevValue + 1?

Reply:
Because `counter.inc()` returns the previous value

The current value can be recovery with `counter.peek()`

If you run this code
```ts (wing)
bring cloud;

let c = new clount.Counter();

let var x = c.inc();
log("${x}"); // print 0
log("${c.peek()}"); //print 1
```